### PR TITLE
Fixes for language parameter in various CLST webservices

### PR DESCRIPTION
**Pull request by proycon**
(automatically reharvested)

### CODE REVIEW

File: tools/Automatic Transcription of Dutch Speech Recordings (Wav file).json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "topic": "OH"
     },
     "mimetypes": [
-        "audio/vnd.wave"
+        "audio/wav"
```
Comment (emanueldima) [inline tools/Automatic Transcription of Dutch Speech Recordings (Wav file).json:27]: You should probably keep the old version, see https://github.com/clarin-eric/switchboard-tool-registry/pull/79. See also this: https://projects.softwareplumbers.com/document-management/tika/commit/bada130efe32560a5b3c26472029f3bd536040b8.

**Comment by proycon** [comment]
I see yes, I will change it in our webservices then, do a new deployment, and reharvest them. (might take a day)

Though I can imagine such issues may arise more frequently, perhaps it's also an idea to implement in the switchboard itself some kind of groups of mimetypes that are considered identical for matching purposes?

**Comment by emanueldima** [comment]
I think you're right, I'm not happy with how the mediatypes are currently used. 

https://github.com/clarin-eric/switchboard/issues/160

**Comment by emanueldima** [comment]
I restored the wave mimetype that Apache Tika recognises, we can now merge this.

**Pull request merged**